### PR TITLE
[Term Entry] python Built-in Function: getattr() 

### PR DESCRIPTION
**Pull request by akiraafu**
### Description

- Added the getattr() term for Python
- Closes issue #2142 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/getattr/getattr.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+The built-in **getattr()** function returns the value of the named property in a specified object/[Class](https://github.com/Codecademy/docs/blob/main/content/python/concepts/classes/classes.md).
+
+## Syntax
+
+```py
+getattr(object, name, default)
+```
+
+The `getattr()` method takes at least 2 parameters: `object`, the object whose attribute we need to work with. `name`, a string with the name of the attribute.
+The `default` parameter is optional, it will be returned when the named attribute does not exist.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/getattr/getattr.md:28]:
```suggestion
- `default`, (optional) a value to be returned when the named attribute does not exist
```

File: content/python/concepts/built-in-functions/terms/getattr/getattr.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Example
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/getattr/getattr.md:30]:
```suggestion
## Examples
```

File: content/python/concepts/built-in-functions/terms/getattr/getattr.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+### Example 1
+
+```py
+class Cat:
+    hobby = 'sleep'
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/getattr/getattr.md:36]:
```suggestion
    hobby = 'sleeping'
```

Comment (caupolicandiaz) [reply content/python/concepts/built-in-functions/terms/getattr/getattr.md:36]: please change this attribute in the other examples as well

**Comment by akiraafu** [comment]
@caupolicandiaz Hi, thanks for the review, I've finished making changes.

**Review by caupolicandiaz — APPROVED** [review]
@akiraafu There were a few small edits that I added. It's ready now for a second review.

**Review by SSwiniarski — APPROVED** [review]
@akiraafu. I did a review and it looks good to merge.

**Comment by SSwiniarski** [comment]
@akiraafu Congrats on the new entry! Live link is here: https://www.codecademy.com/resources/docs/python/built-in-functions/getattr

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

Note: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**